awx/main/models/jobs.py

```diff
@@ -666,6 +666,14 @@ def skipped_hosts(self):
     def processed_hosts(self):
         return self._get_hosts(job_host_summaries__processed__gt=0)
 
+    @property
+    def ignored_hosts(self):
+        return self._get_hosts(job_host_summaries__ignored__gt=0)
```

Comment:
Copying comment from dm w/ @rebeccahhh:

Since the new fields were introduced in 2.8 and older versions of ansible are still supported [1] wondering if this might create issues for us w/ older versions of ansible (guessing not, but seems worth double-checking)

[1] https://access.redhat.com/articles/3382771, log in to https://access.redhat.com/ to see full matrix

Reply:
@rebeccahhh and I walked through this - the job host summaries (including the new fields) default to zero. the way we're accessing the job host summaries here won't cause anything to blow up if ansible < 2.8. looks good!

Reply:
checked with 2.7 and it's :+1: